crates/router/src/connector/opayo.rs

```diff
@@ -87,7 +88,7 @@ impl ConnectorCommon for Opayo {
             .change_context(errors::ConnectorError::FailedToObtainAuthType)?;
         Ok(vec![(
             headers::AUTHORIZATION.to_string(),
-            auth.api_key.into_masked(),
+            auth.api_key.peek().to_string().into_masked(),
```

Comment:
Can you use `expose()` instead?

```suggestion
            auth.api_key.expose().into_masked(),
```

You'll have to import `ExposeInterface` by the way.